lib/pacemaker/libpacemaker_private.h

```diff
@@ -16,6 +16,21 @@
 
 #include <crm/pengine/pe_types.h> // pe_action_t, pe_node_t, pe_working_set_t
 
+// Flags to modify the behavior of the add_colocated_node_scores() method
+enum pcmk__coloc_select {
+    // Apply all relevant colocations
+    pcmk__coloc_select_any          = 0,
+
+    // Apply "this with" colocations instead of "with this" colocation
```

Comment:
This comment implies that "with this" is the default. Does that mean `pcmk__coloc_select_any` applies only "with this" colocations, or does it apply all colocations?

Reply:
Correct, the default applies only "with this" colocations. No good name for that one ... `pcmk__coloc_select_none` or `pcmk__coloc_select_all` would be clearly wrong, and `pcmk__coloc_select_this_with` would make `pcmk__coloc_select_this_with|pcmk__coloc_select_with_this` confusing. Maybe `pcmk__coloc_select_default`? Or maybe even just use 0?

Reply:
I'm rather indifferent about the name; mostly, I'd like the comment to be clearer. It's counterintuitive that "all relevant colocations" doesn't include "this with" colocations.

`pcmk__coloc_select_default` sounds like a good option to me. I don't think `pcmk__coloc_select_this_with|pcmk__coloc_select_with_this` is terribly confusing; kinda have to squint at it, but it tells you explicitly what it's doing. OTOH, it becomes less clear that that's the default behavior.

Reply:
Updated commit to go with `pcmk__coloc_select_default` and improve comment

BTW `pcmk__coloc_select_this_with|pcmk__coloc_select_with_this` would be confusing because it *doesn't* say what it's doing ... it selects only "with this" colocations

Reply:
Oh, yeah that would be very confusing. Thanks!